components/app/myrw/widgets/pages/new.js

```diff
@@ -162,15 +153,14 @@ class WidgetsNew extends React.Component {
     });
   }
 
-  handleChange(value) {
+  handleChange = (value) => {
     const newWidgetObj = Object.assign({}, this.state.widget, value);
     this.setState({ widget: newWidgetObj });
+    this.loadDatasets();
```

Comment:
This change shouldn't be in this file. Also, before anything... Could you check if with https://github.com/resource-watch/resource-watch/pull/1000 the problem is solved?